iphone/Classes/MediaModule.m

```diff
@@ -1105,8 +1106,20 @@ - (void)openPhotoGallery:(id)args
 {
   ENSURE_SINGLE_ARG_OR_NIL(args, NSDictionary);
   ENSURE_UI_THREAD(openPhotoGallery, args);
-  [self showPicker:args isCamera:NO];
+
+  NSArray *types = (NSArray *)[args objectForKey:@"mediaTypes"];
+#if IS_SDK_IOS_14
+  if ([TiUtils isIOSVersionOrGreater:@"14.0"] && ([args objectForKey:@"selectionLimit"] || [TiUtils boolValue:[args objectForKey:@"allowMultiple"] def:NO])) {
```

Comment:
I think the `selectionLimit` property should be **ignored** unless `allowMultiple` is set `true`. The reason is because `selectionLimit` is not supported on Android... so you can't just set that property alone. It wouldn't be portable.

So, I think the "if" condition should be this...
```objc
if ([TiUtils isIOSVersionOrGreater:@"14.0"] && [TiUtils boolValue:[args objectForKey:@"allowMultiple"] def:NO]) {
   // Show multiple selection window.
} else {
   // Show single selection window.
}
```

Reply:
Agree! Updated.